components/ProfileSettings/ProfileSettings.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { Button } from "@mui/material";
+import Menu from "@mui/material/Menu";
+import MenuItem from "@mui/material/MenuItem";
+import styles from "../ProfileSettings/ProfileSettings.module.css";
+import Divider from "@mui/material/Divider";
+import { useRouter } from "next/router";
+import { signOut } from "../../firebase/auth";
+
+const ProfileSettings: React.FunctionComponent = () => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const open = Boolean(anchorEl);
+  const router = useRouter();
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+  const handleSignOut = () => {
+    signOut();
+    router.push("/");
+  };
+  return (
+    <div>
+      <Button
+        variant="contained"
+        className={styles.button}
+        onClick={handleClick}
+      >
+        Cindy ▼
```

Comment:
profile name in corner is all caps. I think u can use textTransform to fix that.